core/src/main/java/org/apache/carbondata/core/datamap/dev/DataMap.java

```diff
@@ -70,4 +70,8 @@ void init(DataMapModel dataMapModel)
    */
   void finish();
 
+  /*
+  * Returns number of records information that are stored in datamap.
+  * */
+  int getNumberOfEntries();
```

Comment:
Add comment here the purpose of this number here.

Reply:
done